### PR TITLE
 Add grpc-web support

### PR DESCRIPTION
**Pull request by heartless-clown**
Initial grpc-web support with updated examples and readme.
Additionally allows to use executable proto dependencies without wrappers.

### CODE REVIEW

File: pkg/rule/rules_nodejs/grpc_web_js_library.go [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+const (
+	grpcWebJsLibraryRuleName   = "grpc_web_js_library"
+	grpcWebJsLibraryRuleSuffix = "_grpc_web_js_library"
+)
+
+func init() {
+	protoc.Rules().MustRegisterRule("stackb:rules_proto:grpc_web_js_library", &grpcWebJsLibrary{})
+}
+
+// grpcWebJsLibrary implements LanguageRule for the 'grpc_web_js_library' rule from @rules_proto.
```
Comment (pcj) [inline pkg/rule/rules_nodejs/grpc_web_js_library.go:21]: Comment needs updating

File: pkg/rule/rules_nodejs/grpc_web_js_library.go [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+// KindInfo implements part of the LanguageRule interface.
+func (s *grpcWebJsLibrary) KindInfo() rule.KindInfo {
+	return jsLibraryKindInfo
+}
+
+// LoadInfo implements part of the LanguageRule interface.
+func (s *grpcWebJsLibrary) LoadInfo() rule.LoadInfo {
+	return rule.LoadInfo{
+		Name:    "@build_stack_rules_proto//rules/nodejs:grpc_web_js_library.bzl",
```
Comment (pcj) [inline pkg/rule/rules_nodejs/grpc_web_js_library.go:37]: the directory `rules/nodejs` implies that the rule implementation is related to `bazelbuild/rules_nodejs`...  is it?

File: plugin/grpc/grpc-web/BUILD.bazel [hunk 3]
```diff
@@ -0,0 +1,17 @@
+load("@build_stack_rules_proto//rules:proto_plugin.bzl", "proto_plugin")
+
+proto_plugin(
+    name = "protoc-gen-grpc-web",
+    tool = select({
+        "@bazel_tools//src/conditions:darwin_x86_64": "@com_github_grpc_grpc_web_releases_download_v1_3_1_protoc_gen_grpc_web_1_3_1_darwin_x86_64//file",
```
Comment (pcj) [inline plugin/grpc/grpc-web/BUILD.bazel:6]: is there a darwin_amd64 option?

File: pkg/rule/rules_nodejs/grpc_web_js_library.go (hunk 2 above)
Comment (pcj) [reply pkg/rule/rules_nodejs/grpc_web_js_library.go:37]: edit: looks like it is: `load("@build_bazel_rules_nodejs//:index.bzl", "js_library")`

**Comment by pcj** [comment]
This looks good, nice figuring things out in a new repo!  A few nits, but otherwise looks ready to merge soon.

**Review by pcj — CHANGES_REQUESTED** [review]

File: pkg/rule/rules_nodejs/grpc_web_js_library.go (hunk 1 above)
Comment (pcj) [reply pkg/rule/rules_nodejs/grpc_web_js_library.go:21]: `from @rules_proto`

**Comment by heartless-clown** [comment]
The CI should be happy now, and hopefully you satisfied with more clear comments in 
[pkg/rule/rules_nodejs/grpc_web_js_library.go](https://github.com/stackb/rules_proto/pull/252/commits/5352aaf306ff87d3e15431c999dc80621da364e4#diff-57a8f064668c061f091c64f8d2b8b440237645fac69af51ea0504c4bec1c686b)

**Review by pcj — APPROVED** [review]

**Comment by pcj** [comment]
Thanks @heartless-clown, merging this.

**Pull request merged**